diff --git a/Dockerfile b/Dockerfile
index 3fa734d..1229df6 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -28,7 +28,7 @@ COPY . /app/
 # Uncomment SECRET_KEY_BASE, RAILS_ENV, and [optionally] RAILS_SERVE_STATIC_FILES for production:
 # ENV SECRET_KEY_BASE=[VALUE OF `bundle exec rake secret`]
 #
-# ENV RAILS_ENV=production
+ENV RAILS_ENV=production
 #
 # ENV RAILS_SERVE_STATIC_FILES=true
 # Leave RAILS_SERVE_STATIC_FILES commented if Nginx/Apache will serve static files instead of rails.
diff --git a/config/environments/production.rb b/config/environments/production.rb
index 1a5ec07..a4e05d8 100644
--- a/config/environments/production.rb
+++ b/config/environments/production.rb
@@ -1,7 +1,7 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
-  config.force_ssl = true
+  config.force_ssl = false
 
   # Code is not reloaded between requests.
   config.cache_classes = true
